src/codeu/chat/client/core/UserContext.java

```diff
@@ -53,4 +56,18 @@ public ConversationContext start(String name) {
 
     return all;
   }
+  
+
+  // USERstatusUpdate()
+  public HashMap<Uuid, ArrayList<ArrayList<String>>> USERstatusUpdate(){ 
+	  HashMap<Uuid, ArrayList<ArrayList<String>>> userUpdates = this.user.USERstatusUpdate(this.conversations(), controller.statusUpdate()); 
```

Comment:
Since there is no other opertations performed on userUpdate, maybe just `return this.user.USERstatusUpdate(this.conversations(), controller.statusUpdate());`